Header header
string              name
geometry_msgs/Point position_person
petra/Leg           leg1
petra/Leg           leg2
float64             reliability
